modules/nixos/where-is-my-sddm-theme.nix

```
@@ -0,0 +1,93 @@
+{ catppuccinLib }:
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  inherit (builtins) elem;
+  inherit (catppuccinLib) importINI;
+  inherit (lib)
+    literalMD
+    mkOption
+    recursiveUpdate
+    types
+    ;
+
+  cfg = config.catppuccin.where-is-my-sddm-theme;
+  enable = cfg.enable && config.services.displayManager.sddm.enable;
+in
+
+{
+  options.catppuccin.where-is-my-sddm-theme =
+    catppuccinLib.mkCatppuccinOption {
+      name = "where-is-my-sddm-theme";
```

```suggestion
      name = "where-is-my-sddm-theme";
      default = config.catppuccin.sddm.enable || config.catppuccin.enable;
      defaultText = "config.catppuccin.sddm.enable || config.catppuccin.enable";
```
I think this will fix the conflict with our base SDDM module